config/src/config/network_config.rs

```diff
@@ -79,6 +83,10 @@ pub struct NetworkConfig {
     pub mutual_authentication: bool,
     pub network_id: NetworkId,
     pub runtime_threads: Option<usize>,
+    pub inbound_rx_buffer_size_bytes: Option<u32>,
+    pub inbound_tx_buffer_size_bytes: Option<u32>,
+    pub outbound_rx_buffer_size_bytes: Option<u32>,
+    pub outbound_tx_buffer_size_bytes: Option<u32>,
```

Comment:
I heard `Option`s are not friendly to terraform templating, etc, shall we just default to the default numbers?

Reply:
@rustielin  @sherry-x ?

Reply:
No it should be fine, so long as this lands: https://github.com/aptos-labs/aptos-core/pull/4847